tensorflow/lite/toco/README.md

```diff
@@ -8,8 +8,8 @@ the usage documentation.
 
 Usage information is given in these documents:
 
-*   [Command-line glossary](../g3doc/convert/cmdline_reference.md)
-*   [Command-line examples](../g3doc/convert/cmdline_examples.md)
+*   [Command-line glossary](../g3doc/r1/convert/cmdline_reference.md)
+*   [Command-line examples](../g3doc/r1/convert/cmdline_examples.md)
 *   [Python API examples](../g3doc/convert/python_api.md)
 
 ## Where the converter fits in the TensorFlow landscape
```

Comment:
The line including the drawing (line 29) needs to link to here: 

https://github.com/tensorflow/tensorflow/tree/3cfbf14e80a4b5feb9e1a786e02ff705b42f83ef/tensorflow/lite/g3doc/r1/images/convert (basically within the r1 directory instead of the main directory)

Reply:
Fixed in 994b6defd9778eb1478977b8249a458bc35cb523.